test/view_model_tests/after_auth_view_model_tests/add_post_view_model_test.dart

```diff
@@ -67,25 +66,25 @@ void main() {
 
       verify(notifyListenerCallback());
     });
-    test("Check if uploadPost() is working fine", () {
-      final notifyListenerCallback = MockCallbackFunction();
-      final model = AddPostViewModel()..addListener(notifyListenerCallback);
-      model.initialise();
-
-      when(navigationService.showSnackBar("Post is uploaded"))
-          .thenAnswer((_) {});
-
-      model.uploadPost();
-
-      expect(model.imageFile, null);
-      verify(notifyListenerCallback());
-      expect(model.controller.text, "");
-      verify(
-        navigationService.showTalawaErrorSnackBar(
-          "Post is uploaded",
-          MessageType.info,
-        ),
-      );
-    });
+    // test("Check if uploadPost() is working fine", () {
+    //   final notifyListenerCallback = MockCallbackFunction();
+    //   final model = AddPostViewModel()..addListener(notifyListenerCallback);
+    //   model.initialise();
+
+    //   when(navigationService.showSnackBar("Post is uploaded"))
+    //       .thenAnswer((_) {});
+
+    //   model.uploadPost();
+
+    //   expect(model.imageFile, null);
+    //   verify(notifyListenerCallback());
+    //   expect(model.controller.text, "");
+    //   verify(
+    //     navigationService.showTalawaErrorSnackBar(
+    //       "Post is uploaded",
+    //       MessageType.info,
+    //     ),
+    //   );
+    // });
```

Comment:
Please remove it if its not needed later.

Reply:
@Kushalrock Please make the requested updates

Reply:
@palisadoes @noman2002 I have done so you guys can approve it for now. Also, please look at issue #1680 and edit it to the test format. But yeah please don't add the good first issue label as this is not something to tackle for someone with beginner flutter experience instead someone who knows a bit more about codebase. I may take up the issue